cmake_templates/zeek-config.in

```diff
@@ -18,6 +21,19 @@ version="@ZEEK_VERSION_FULL@"
 zeek_dist="@ZEEK_DIST@"
 zeekpath="@DEFAULT_ZEEKPATH@"
 
+report_feature() {
+    # $1: test result
+    if [ "$1" = "yes" ] || [ "$1" = "true" ] || [ "$1" = "ON" ]; then
+        echo "yes"
+        exit 0
+    else
+        echo "no"
+        exit 1
+    fi
+
+    # Won't return
```

Comment:
The comment would become redundant if you'd get rid of the `else`, e.g.,

```suggestion    fi

    echo "no"
    exit 1
```

Reply:
Done